build.gradle

```diff
@@ -7,10 +7,13 @@ buildscript {
     dependencies {
         classpath 'gradle.plugin.com.palantir:gradle-circle-style:1.1.2'
         classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.4'
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.3'
-        classpath 'com.netflix.nebula:gradle-dependency-lock-plugin:4.9.5'
-        classpath 'com.netflix.nebula:nebula-dependency-recommender:6.0.0'
-        classpath 'com.netflix.nebula:nebula-publishing-plugin:5.1.5'
+        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.3', {
+            // This brings in maven 2, conflicts with our maven dependencies from gradle/nebula-dependency-recommender
+            exclude module: 'maven-ant-tasks'
```

Comment:
This is sad that we have to do this, but a dep of this guy, namely `org.codehaus.groovy.modules.http-builder:http-builder`, brings in this old library which no longer exists in maven 3.

Since both nebula dependency recommender and gradle use maven 3, this dependency's presence was causing runtime JVM errors when calling `netflix.nebula.dependency.recommender.provider.MavenBomRecommendationProvider#getRecommendations`.

It seems I can run bintrayUpload (at least locally) without this and it doesn't crash in an unexpected way, so maybe it's fine.

```
+--- com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.3
|    +--- org.codehaus.groovy.modules.http-builder:http-builder:0.7.2
|    |    +--- org.apache.httpcomponents:httpclient:4.2.1 -> 4.3.6
|    |    |    ...
|    |    +--- net.sf.json-lib:json-lib:2.3
|    |    |    ...
|    |    +--- net.sourceforge.nekohtml:nekohtml:1.9.16
|    |    |    ...
|    |    \--- xml-resolver:xml-resolver:1.2
|    \--- org.apache.maven:maven-ant-tasks:2.1.3
```